keras_cv/layers/object_detection/nms_prediction_decoder.py

```diff
@@ -89,28 +89,33 @@ def _decode_box_predictions(self, anchor_boxes, box_predictions):
         )
         return boxes
 
-    def call(self, images, predictions):
+    def call(self, images, predictions, anchor_boxes=None):
```

Comment:
one more comment here: isn't the prediction a dict of boxes, classes and scores? or do we re-concatenate it somewhere else in the model?